content/docs/setup/listeners/sni.md

```diff
@@ -0,0 +1,567 @@
+---
+title: SNI
```

Comment:
add as a card to the listener index, such as here: https://0b611056.kgateway-io.pages.dev/docs/setup/listeners/#guides